src/components/Header/Header.scss

```diff
@@ -1,36 +1,34 @@
 $headerHeightDesktop: 160px;
 $headerHeightMobile: 64px;
+
+#header-height {
+  @media screen and (max-width: 480px) {
+    height: $headerHeightMobile;
+  }
+}
+
+#title-margin {
+  @media screen and (max-width: 480px) {
+    margin-left: $headerHeightMobile;
+  }
+}
```

Comment:
``` suggestion
  @media screen and (max-width: 480px) {
      .header{
          height: $headerHeightMobile;
          img {
                $headerHeightMobile;
          } 
      }
  }
```
What do you think about something like that?

Reply:
Nice João, i liked this way, i didn't know i could do this inside the "@media", thanks!!! Already made the changes